src/SystemCommands-RefactoringSupport/SycRefactoringPreview.class.st

```diff
@@ -257,9 +257,12 @@ SycRefactoringPreview >> updateChanges [
 SycRefactoringPreview >> updateTablePresenter [
 	| aCompositeChange |
 	aCompositeChange := RBCompositeRefactoryChange new.
+	aCompositeChange onSystemDictionary: 
+		(changes at:1 ifAbsent: [ self error: 'No changes to apply' ]) model environment. 
```

Comment:
This is strange.
Why not `changes ifEmpty:`?

Why the validation should be here?
I have the impression that before this worked even without changes, and now you added a validation so if there are changes, this fails.
Is that ok?